SpatialGDK/Source/SpatialGDK/Private/Tests/TestRoutingWorker.cpp

```diff
@@ -204,7 +204,8 @@ class ConnectionHandlerStub : public AbstractConnectionHandler
 		check(!HasNextOpList());
 		TArray<TArray<OpList>> NewListsOfOpLists;
 		TArray<OpList> OpLists;
-		OpLists.Add(Builder.Move().CreateOpList());
+		OpLists.Add(MoveTemp(Builder).CreateOpList());
+		Builder = EntityComponentOpListBuilder();
```

Comment:
Curious to know why this function overrides the provided builder?

Reply:
Because the builder is now moved from.